deploy/kubernetes/helm/che/values.yaml

```diff
@@ -67,7 +67,7 @@ global:
     fsGroup: 1724
   postgresDebugLogs: false
 
-#che:
+che: {}
```

Comment:
@monaka Can not we do the same for workspace section?
If `workspace.devfileRegistryUrl` field is uncommented, should `{}` be removed?

Reply:
@sleshchenko Yes. `{}` must be removed then. This approach is often shown on charts in the official Helm repository. (Just one example, Traefik https://github.com/helm/charts/blob/4cbc02a65f7c1bfef4cd4b03e3556a289b2f2014/stable/traefik/values.yaml#L35-L39)